src/pages/landing/[landingPage]/index.tsx

```diff
@@ -0,0 +1,146 @@
+    }
+  }[]
+}
+
+const LandingPage = ({
+  landingPage,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const { pageTitle, pageDescription, documents, collections, articles } =
+    landingPage
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const pageContent = (): any => {
+    return (
+      <>
+        <h1 className={styles.pageTitle}>{pageTitle}</h1>
+        <p className={styles.pageDescription}>{pageDescription}</p>
+
+        {documents.length >= 1 && <h2 id="documents">Documents</h2>}
+        {documents.length >= 1 && (
+          <div>
+            {documents.map((section: DocumentsType, index: number) => {
+              return (
+                <Accordion
+                  key={`${index}__${section}`}
+                  bordered
+                  items={[
+                    {
+                      title: section.title,
+                      expanded: false,
+                      id: `${section}__${index}`,
+                      headingLevel: 'h3',
+                      content: (
+                        <div className={styles.accordionContent}>
+                          {section.document.map((doc, index) => {
+                            return (
+                              <p key={`${doc.title}__${index}`}>
+                                <a href={doc.file.url}>{doc.title}</a>
+                              </p>
+                            )
+                          })}
+                        </div>
+                      ),
+                    },
+                  ]}
+                />
+              )
+            })}
+          </div>
+        )}
+
+        {collections.length >= 1 && <h2 id="collections">Collections</h2>}
+
+        {collections.length >= 1 && (
+          <div className={styles.collectionContainer}>
+            {collections.map((collection: CollectionRecord, index: number) => {
+              return (
+                <Grid
+                  key={`${index}__${collection}`}
+                  tablet={{ col: 6 }}
+                  desktop={{ col: 4 }}>
+                  <Collection title={collection.title}>
+                    {collection.bookmarks.map(
+                      (bookmark: CMSBookmark, index: number) => {
+                        return (
+                          <Bookmark
+                            key={`${index}__${bookmark.label}`}
+                            bookmarkDescription={bookmark.description}
+                            href={bookmark.url}>
+                            {bookmark.label}
+                          </Bookmark>
+                        )
+                      }
+                    )}
+                  </Collection>
+                </Grid>
+              )
+            })}
+          </div>
+        )}
+
+        {articles.length >= 1 && <h2 id="articles">Articles</h2>}
+        {articles.length >= 1 && (
+          <ArticleList articles={articles} landingPage={true} />
+        )}
```

Comment:
Unfortunately, I had to do a bit of prop drilling here. The `landingPage` prop gets passed down to `ArticleListItem` and is used to link to the dynamic path for an article that is related to a landing page. I did this to save a bit of time on what would probably need to be a larger refactor. Let me know if you see a better solution!